src/blockchain/milestones.js

```diff
@@ -34,8 +34,13 @@ const getDonationPaymentsByToken = donations => {
 const createPaymentConversationAndSendEmail = async ({ app, milestone, txHash }) => {
   try {
     const milestoneId = milestone._id;
-    const { recipient, campaignId, title } = milestone;
+    const { recipient, campaignId, title, owner } = milestone;
 
+    // When  recipient is an address that doesnt have account in Giveth recipient is null
+    // so below checking is for this purpose to send emails for milestone's owner instead of recipient
+    const email = (recipient && recipient.email) || owner.email;
```

Comment:
As #315 says the requirement is the campaign owner (not milestone owner) should receive email when the milestone recipient is the campaign.

Reply:
Yes Amin I was wrong
For campaign owner the code doesn't come  here, it's just for milestones that have anonymous user recipient

Reply:
@mohammadranjbarz I am not sure we should email to milestone owner, or if we do should do always (not only when recipient is anonymous)
@horus2021 what do you think?

Reply:
@aminlatifi Now I undo sending email for owner if recipient is null